README.markdown

```diff
@@ -53,6 +53,12 @@ Scala-Forklift supports both Slick 3.1 and Slick 3.2. The latest versions of Sca
 | 2.11.x         | `3.1.x`       | `libraryDependencies += "com.liyaos" %% "scala-forklift-slick" % "0.2.3"` |
 | 2.12.x, 2.11.x | `3.2.x`       | `libraryDependencies += "com.liyaos" %% "scala-forklift-slick" % "0.3.1"` |
 
+You might have to add these resolvers to your build.sbt file:
+```
+resolvers += Resolver.sonatypeRepo("snapshots")
```

Comment:
Hi, thanks for the contribution!

If I have understood sonatypeRepo correctly, you should only need to add this resolver if you are using the snapshot version of scala-forklift, right? Or did I miss something?

If it is the case that one only needs to add this if they wants to use the snapshot version, can you modify the wording here to state that?